powersimdata/network/europe_tub/model.py

```diff
@@ -0,0 +1,140 @@
+import os
+import shutil
+from zipfile import ZipFile
+
+from powersimdata.network.constants.region.europe import (
+    abv2country,
+    abv2timezone,
+    interconnect2abv,
+)
+from powersimdata.network.helpers import (
+    check_and_format_interconnect,
+    interconnect_to_name,
+)
+from powersimdata.network.model import ModelImmutables
+from powersimdata.utility.helpers import _check_import
+
+pypsa = _check_import("pypsa")
+zenodo_get = _check_import("zenodo_get")
+
+
+class TUB:
+    """PyPSA Europe network.
+
+    :param str/iterable interconnect: interconnect name(s).
+    :param int reduction: reduction parameter (number of nodes in network). If None,
+        the full network is loaded.
+    :param bool overwrite: the existing dataset is deleted a new dataset is downloaded
+        from zenodo.
```

Comment:
`the existing dataset is deleted a new dataset is downloaded` -> `the existing dataset is deleted and a new dataset is downloaded`?

Reply:
Done